.gitignore

```diff
@@ -22,3 +22,7 @@ terraform*
 .env.production.local
 
 npm-debug.log*
+
+# emacs backup files
```

Comment:
What's this for?

Reply:
Emacs backup files end in ~. Adding *~ improves the workflow for emacs users